README.md

```diff
@@ -226,6 +226,58 @@ You will see this result in the response to your `curl` call to the predict endp
 
 Now you've seen how easy it can be to serve a deep learning model with TorchServe! [Would you like to know more?](docs/server.md)
 
+### Get metrics
```

Comment:
Move this section to `metric_api.md` under docs and add a reference in `docs/rest_api.md`

Reply:
Done